etor/src/main/java/gov/hhs/cdc/trustedintermediary/etor/EtorDomainRegistration.java

```diff
@@ -81,14 +85,26 @@ public Map<HttpEndpoint, Function<DomainRequest, DomainResponse>> domainRegistra
 
         if (ApplicationContext.getEnvironment().equalsIgnoreCase("local")) {
             ApplicationContext.register(OrderSender.class, LocalFileOrderSender.getInstance());
-            ApplicationContext.register(
-                    PartnerMetadataStorage.class, FilePartnerMetadataStorage.getInstance());
+
```

Comment:
Can we simplify the logic below, and the missing azurestorage instance like this:
```
ApplicationContext.register(AzureClient.class, AzureClient.getInstance());

if (ApplicationContext.getProperty("DB_URL") != null) {
    ApplicationContext.register(SqlDriverManager.class, EtorSqlDriverManager.getInstance());
    ApplicationContext.register(DbDao.class, PostgresDao.getInstance());
    ApplicationContext.register(PartnerMetadataStorage.class, DatabasePartnerMetadataStorage.getInstance());
} else if (ApplicationContext.getEnvironment().equalsIgnoreCase("local")) {
    ApplicationContext.register(PartnerMetadataStorage.class, FilePartnerMetadataStorage.getInstance());
} else {
    ApplicationContext.register(PartnerMetadataStorage.class, AzureStorageAccountPartnerMetadataStorage.getInstance());
}

if (ApplicationContext.getEnvironment().equalsIgnoreCase("local")) {
    ApplicationContext.register(OrderSender.class, LocalFileOrderSender.getInstance());
} else {
    ApplicationContext.register(OrderSender.class, ReportStreamOrderSender.getInstance());
}
```